src/main/resources/application.yml

```diff
@@ -145,7 +145,7 @@ doc_assembly:
   oconFormOrganisationWithoutDQs: "CV-CMC-GOR-ENG-0015.docx"
   noticeOfTransferToCcbcSentToDefendantTemplateId: "CV-CMC-GOR-ENG-0016.docx"
   paperResponseAdmissionTemplateId: "CV-CMC-GOR-ENG-0016.docx"
-  defendantPinLetterTemplateID: "CV-CMC-LET-ENG-00630.docx"
+  defendantPinLetterTemplateID: "CV-CMC-LET-ENG-00630-1.odt"
```

Comment:
Why the doc name is .odt

Reply:
As discussed, this will be handled via [CIV-7013](https://tools.hmcts.net/jira/browse/CIV-7013)